server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -128,10 +128,10 @@ public void unregister(CommandHandler handler) {
     }
 
     /**
-     * Verifies if the command can be posted to this {@code CommandBus}.
+     * Verifies that the command message can be posted to this {@code CommandBus}.
      *
-     * <p>A command can be posted if it has either dispatcher or handler registered with
-     * the command bus.
+     * <p>A command message can be posted if it has either dispatcher or handler registered with
```

Comment:
We do not post message, we post a whole command. We associate dispatchers or messages with types of command messages.